src/scheduler/data_types.h

```diff
@@ -738,6 +739,7 @@ struct node_info
 	int bucket_ind;			/* index in server's bucket array */
 	int node_ind;			/* node's index into sinfo->unordered_nodes */
 	node_partition **np_arr;	/* array of node partitions node is in */
+	int server_sock;		/* server socket associated with this node */
```

Comment:
We might not need the node socket (and we don't need it right now).  We always talk to the job's server.  It might be true the job might move to another server, but the scheduler talks to the server which owns the job now.  We do need some way to match server to node, but that might be done via name and not socket.

Reply:
Yes @bhroam. As per the updated design doc we don't need server_sock. We are actually getting server's fd by using server_instance_id which actually consists of "server_instance_name:port". I request you to go through the updated design document.

Reply:
Ahh, the design has indeed updated since I last reviewed it.  I'd rename this from server_sock, to svr_inst_id since it is not a socket.